README.md

```diff
@@ -25,82 +56,70 @@ The API version is specified through the accept header, as follows:
 
-see [Rswag readme](https://github.com/rswag/rswag/blob/master/README.md) for initial setup and/or modifications.
+Attached to each summary object are the input data, represented as one or more sub-objects, representing the individual items/transactions: the `GrossIncomeSummary` has income of various types, `CapitalSummary` has capital items, and the `DisposableIncomeSummary` various outgoings.
 
-The `swagger` folder in the root directory has one `swagger.yaml` within a version number folder - e.g. `swagger/v4/swagger.yaml`. This file is what defines the swagger ui page displayed at `/api-docs`. This file is generated using rswag's rake task - `rake rswag:specs:swaggerize`.
+Each of the three summary objects also has a number of "Eligibility" objects - one for each proceeding type specified on the assessment.  Each eligibility object contains the upper and lower thresholds for that proceeding type, and will eventually hold the result for that proceeding type.
 
-The `swagger.yaml` file that is generated is defined by a combination of "global" settings in `spec/swagger_helper.rb` and indivual spec files that are, by our convention, stored in `spec/requests/swagger_docs/<version>/*.spec.rb`.
 
-You can generate a new endpoint spec file using:
-```sh
-rails generate rspec:swagger MyController
-```
+## Logic flow of the calculations
 
-You can update an existing endpoint by modifying it's spec and then running:
-```sh
-rake rswag:specs:swaggerize
-```
+The `AssessmentController` calls the `MainWorkflow`, which immediately branches off to the `PassportedWorkflow` or `UnpassportedWorkflow`.  The main difference is that unpassported applications are assessed on capital, gross income and disposable income, whereas passported applications are only assessed on capital.
+
+In each case, the workflow takes each of the assessments in turn, calls a collator to look at all the sub-objects under the summary and come up with a total figure in the case of capital, or a monthly equivalent figure in the case of gross or disposable  income, and these results are stored on the associated summary object.  After collation, an assessor is called for each summary which stores the results (eligible, ineligible, contribution required) in each of the eligibility objects (one for each proceeding type).  Finally, the main assessor is called to work out the overall result for each proceeding type taking into account the results from the capital, gross income and disposable income assessments.  The assessments controller then calls the main decorator to format the output in the required structure to send back to the client.
+
+# Development
 
 ## Setting the env vars
-To run the integration tests you will need to set up a `.env` file in the root folder.
-
-It should contain the following values:
-```shell script
-GOOGLE_SHEETS_PRIVATE_KEY_ID
```

Comment:
have we lost this section about the 4 fields? Think we still need it

Reply:
It is still in the section "Environment variables for Integration tests (spreadsheets)" on L224